test/fulfillmentSpec.js

```diff
@@ -177,6 +178,60 @@ describe('GET /fulfillment', function () {
       .expect(this.executionConditionFulfillment)
       .expect(validator.validateFulfillment)
       .end()
+  }
+
+  it('should return a fulfillment', returnFulfillment)
+
+  describe('when hold account is busy', function () {
+    const ARGS_EXPECTED = {
+      '0': 'UPDATE "L_ACCOUNTS" SET "BALANCE" = "BALANCE" + ? WHERE "NAME" = ?',
+      '1': [ -10, 'hold' ]
+    }
+
+    beforeEach(function () {
+      dbFailureMock.timesQueryShouldFail[JSON.stringify(ARGS_EXPECTED)] = 3
+    })
+
+    afterEach(function () {
+      const timesLeft = dbFailureMock.timesQueryShouldFail[JSON.stringify(ARGS_EXPECTED)]
+      dbFailureMock.timesQueryShouldFail[JSON.stringify(ARGS_EXPECTED)] = 0
+      expect(timesLeft).to.equal(0)
+    })
+
+    it('should return a fulfillment', returnFulfillment)
+
+    describe('for a very long time', function () {
```

Comment:
Since there is only one test under this `describe`, I think it would be better to just have the test by itself and to just put the setup / teardown steps in the test (especially because the after includes an assertion that I think it actually part of the test)